components/script/dom/webidls/HTMLTableElement.webidl

```diff
@@ -6,7 +6,7 @@
 // https://www.whatwg.org/html/#htmltableelement
 interface HTMLTableElement : HTMLElement {
            attribute HTMLTableCaptionElement? caption;
-  HTMLElement createCaption();
```

Comment:
Do you remember why you did this @dzbarsky? For reference: https://html.spec.whatwg.org/multipage/tables.html#htmltableelement